### PR TITLE
add sql style

### PR DESCRIPTION
**Pull request by wendyshaohudl**
Adding no number requirement for alias.

### CODE REVIEW

File: style/hudl_sql_style_guide.md [hunk 1]
```diff
@@ -24,6 +24,7 @@ description: A guide to writing clean, clear, and consistent SQL.
 * Include links to hudl id constants or references to important links
 * All tables and views should adhere to all lowercase, words separated with underscores
 * Always use AS when creating aliases for columns, tables, subqueries
+* No numbers or special charaters in aliases
```
Comment (lucas-smith) [inline style/hudl_sql_style_guide.md]:
```suggestion
* Avoid using numbers or special characters in aliases
```

**Review by lucas-smith — COMMENTED** [review]
Please see comments before merge.

**Review by tjengel — APPROVED** [review]
With Lucas's changes

**Comment by wendyshaohudl** [comment]
Updated based on Lucas's suggestion.

**Pull request merged**